core/domain/prod_validation_jobs_one_off.py

```diff
@@ -3183,12 +3173,34 @@ def _validate_final_reveiwer_id(cls, item):
                 'Entity id %s: Final reviewer id %s is not empty but '
                 'suggestion is in review' % (item.id, item.final_reviewer_id))
 
+    @classmethod
+    def _validate_score_category(cls, item):
+        """Validate that the score_category subtype for suggestions matches the
+        exploration category.
+
+        Args:
+            item: ndb.Model. GeneralSuggestionModel to validate.
+        """
+        if item.target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
+            score_category_sub_type = (
+                item.score_category.split(
+                    suggestion_models.SCORE_CATEGORY_DELIMITER)[1])
+            target_exploration = (
+                exp_fetchers.get_exploration_by_id(item.target_id))
```

Comment:
This should be done using fetching of external instances -- https://github.com/oppia/oppia/blob/develop/core/domain/prod_validation_jobs_one_off.py#L233.

Reply:
done